style.css

```diff
@@ -1,8 +1,292 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+
```

Comment:
For the whole file, remove empty lines and properly format the code using formatters like Prettier.